cfme/tests/services/test_provision_stack.py

```diff
@@ -181,35 +205,45 @@ def create_template(setup_provider, provider, dialog):
 
 
 def prepare_stack_data(provider, provisioning):
-    stackname = "test" + fauxfactory.gen_alphanumeric()
+    stack_name = "test" + fauxfactory.gen_alphanumeric()
```

Comment:
Those dashes will fail on Azure.  I think that's why sshveta made her own.  Don't think we can use that.

return 'test-{}-{}'.format(context, fauxfactory.gen_alphanumeric(length=random_chars).lower())